daemon/k8s_watcher.go

```diff
@@ -233,6 +239,8 @@ func (d *Daemon) EnableK8sWatcher(reSyncPeriod time.Duration) error {
 		return fmt.Errorf("Unable to create cilium network policy client: %s", err)
 	}
 
+	cnpCM = controller.NewManager()
```

Comment:
Not a big deal but this could go with the variable declaration directly. That way it will never be nil

Reply:
Oh you're right. done